.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+maxColumn = 80
+
+style = IntelliJ
```

Comment:
I see 2 things here:
* the formatting rules to be applied to projects - I view this as something that the client ultimately decides (e.g. as @fommil suggests), but it is certainly something we can initially supply a default for (which can easily be modified or simply changed on a per project basis)
* the formatting rules to be applied to this plugin - I view this as a Cake controlled thing, and so can see that this would be different